test/plain-list/AutosuggestApp.test.js

```diff
@@ -81,6 +83,23 @@ describe('Default Autosuggest', () => {
     });
   });
 
+  describe('when input field is focused and empty', () => {
+    beforeEach(() => {
+      focusInput();
+      clearEvents();
+    });
+
+    it('should let browser handle dpad_down', () => {
```

Comment:
These tests look good, but can we leave out the 'dpad' references in the names? That doesn't make much sense without any context, and they're not really the issue.

The problem (and this fix) is about letting the browser handle arrow key events, whether they come from a keyboard or a dpad or whatever else. There's other issues that have tried to do the same thing for similar reasons elsewhere, unrelated to Android TV, like #522 and #602.